babel/dates.py

```diff
@@ -395,7 +395,9 @@ def get_timezone_gmt(datetime=None, width='long', locale=LC_TIME):
 
     :param datetime: the ``datetime`` object; if `None`, the current date and
                      time in UTC is used
-    :param width: either "long" or "short"
+    Added new parameter for width to resolve bug #325
+    :param width: either "long" or "short" or "extra"
```

Comment:
Perhaps `iso8601` would be a better name for this width, for the TR35 document seems to state the goal is to have ZZZZZ be an ISO8601 compliant timezone formatting.